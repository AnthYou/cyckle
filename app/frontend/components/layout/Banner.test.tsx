import React from "react";
import renderer from "react-test-renderer";

import Banner from "./Banner";

describe('<Banner />', () => {
  it('should render correctly', () => {
    const component = renderer.create(<Banner />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  })
});
