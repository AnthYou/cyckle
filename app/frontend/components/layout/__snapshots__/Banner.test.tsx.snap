// Vitest Snapshot v1

exports[`<Banner /> > should render correctly 1`] = `
<div
  className="grid grid-cols-1 sm:grid-cols-2 items-center"
>
  <div
    className="w-full"
  >
    <h1>
      Et si on louait un vélo ?
    </h1>
    <p>
      Louer un vélo entre particuliers n'a jamais été aussi simple.
    </p>
    <div
      className="flex"
    >
      <button
        className="_Button_4ea4ec _primary_4ea4ec mr-2"
      >
        Je souhaite louer
      </button>
      <button
        className="_Button_4ea4ec _secondary_4ea4ec"
      >
        Je suis propriétaire
      </button>
    </div>
  </div>
  <div
    className="relative"
  >
    <img
      alt="Cyclist"
      className="w-full absolute"
      src="/@fs/@/images/biking.svg"
    />
    <img
      alt="Cyclist"
      className="w-full"
      src="/@fs/@/images/blob.svg"
    />
  </div>
</div>
`;
